localization/english/culture/wc_cultural_innovations_l_english.yml

```diff
@@ -59,4 +59,6 @@
  #High Med
  innovation_arkonite:0 "Arkonite"
  innovation_arkonite_desc:0 "Arkonite is used daily in our society. From powering arcane devices and light sources, to helping us channel magics."
- 
\ No newline at end of file
+ #Early Med
+ innovation_storm_hammers:0 "Storm Hammers"
+ innovation_storm_hammers_desc:0 "The craftsmen of the Aeries, masters of their art, imbue the war hammers of their gryphon riders with the power of thunder and lightning, to strike true at their enemies and return to their owners safe and sound."
```

Comment:
I was thinking if this maybe should be regional instead of cultural? Just a thought, no explanation/justification, just a thought. Don't think that there's any lore behind the storm hammer upgrade from WC3, the description is completely made up, too...

Reply:
I think cultural is more appropriate here. Stormhammers are only used by wildhammers and seem to be a major part of their traditions. Also, the description is fairly accurate if we accept WC2 material.

Reply:
I think cultural is more appropriate here. Stormhammers are only used by wildhammers and seem to be a major part of their traditions. Also, the description is fairly accurate if we accept WC2 material.

Reply:
Should this be for wildhammers only, or all cultures that have gryphon riders?